src/components/TransactionDetails/Transfer/Transfer.vue

```diff
@@ -78,5 +78,5 @@ export default class Transfer extends Vue {
 </script>
 
 <style lang="less" scoped>
-@import "../TransactionDetails.less";
```

Comment:
what happens to **common styles** that are put inside TransactionDetails.less ? You should not remove this import.

Reply:
the transfer should has own styles  instead of  the other component styles.

Reply:
and it has already cause some style conflict when i reform the modal style.